test/dataframe.jl

```diff
@@ -296,12 +296,14 @@ module TestDataFrame
 
     #Check the output of unstack
     df = DataFrame(Fish = CategoricalArray{Union{String, Missing}}(["Bob", "Bob", "Batman", "Batman"]),
-                   Key = Union{String, Missing}["Mass", "Color", "Mass", "Color"],
+                   Key = CategoricalArray{Union{String, Missing}}["Mass", "Color", "Mass", "Color"],
```

Comment:
Do we still have similar tests with non-categorical columns?

Reply:
added